instrumentation/activej-http-6.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/activejhttp/ActivejHttpServerUtil.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.activejhttp;
+
+import io.activej.http.HttpError;
+import io.activej.http.HttpHeaders;
+import io.activej.http.HttpRequest;
+import io.activej.http.HttpResponse;
+import io.activej.http.HttpVersion;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import javax.annotation.Nullable;
+
+public final class ActivejHttpServerUtil {
+
+  private ActivejHttpServerUtil() {
+    throw new UnsupportedOperationException();
+  }
+
+  static String getHttpRequestMethod(HttpRequest request) {
+    return request.getMethod().name();
+  }
+
+  static List<String> requestHeader(HttpRequest request, String name) {
+    String headerValue = request.getHeader(HttpHeaders.of(name));
+    if (headerValue == null) {
+      return Collections.emptyList();
```

Comment:
We usually use static import here, for detail, you can refer to https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/contributing/style-guideline.md#static-imports
It had better to check other places and adjust them together.